### PR TITLE
Updated harness docs for yaml URL usage

### PR DESCRIPTION
**Pull request by LukasMcClelland**
Doc update to go with code change PRs:
https://wwwin-github.cisco.com/pyATS/pyats/pull/217
https://wwwin-github.cisco.com/pyATS/genie/pull/140

and accompanying doc PR: 
https://github.com/CiscoTestAutomation/pyats/pull/116

Let's users know that they can use URLs for the following:

--subsection-datafile
--mapping-datafile
--verification-datafile
--trigger-datafile
--config-datafile
--pts-datafile

Admittedly, I didn't go through every page of the docs looking for every reference. The page I edited seemed to be the main arguments page. Please let me know if I need to go through the docs thoroughly and edit more places

### CODE REVIEW

**Review by dwapstra — COMMENTED** [review]
How about health docs?

**Comment by LukasMcClelland** [comment]
dwapstra left a comment 
> 
> 
> How about health docs?

https://wwwin-github.cisco.com/pyATS/genielibs/pull/669 Code change for `--health-file`

File: docs/userguide/harness/user/reference.rst [hunk 1]
```diff
@@ -26,12 +26,12 @@ All arguments are optionals.
     ``--trigger-uids``, "Specifies the list of triggers uids to run (list or logic expression)"
     ``--trigger-groups``, "Specifies the list of triggers groups to run (logic expression)"
     ``--devices``, "List of devices to connect to (in case of no mapping datafile)"
-    ``--subsection-datafile``, "Full path/name to the common_(setup/cleanup) subsection datafile"
-    ``--mapping-datafile``, "Full path/name to the mapping datafile"
-    ``--verification-datafile``, "Full path/name to the verification datafile"
-    ``--trigger-datafile``, "Full path/name to the trigger datafile"
-    ``--config-datafile``, "Full path/name to the config datafile"
-    ``--pts-datafile``, "Full path/name to the PTS datafile"
+    ``--subsection-datafile``, "Full path/name or URL to the common_(setup/cleanup) subsection datafile"
```
Comment (tahigash) [inline docs/userguide/harness/user/reference.rst:29]: just want to confirm. pyATS command's help is also updated?

File: docs/userguide/harness/user/reference.rst [hunk 2]
```diff
@@ -283,6 +285,7 @@ Below you will find additional information on each of those arguments:
     .. code-block:: bash
 
          genie run --trigger-datafile /path/triggerdatafile.yaml
+         genie run --trigger-datafile "http://path.to/triggerdatafile.yaml"
```
Comment (tahigash) [inline docs/userguide/harness/user/reference.rst:288]: `genie run` command is obsolute. let's change to `pyats run job`
```suggestion
         pyats run job --trigger-datafile /path/triggerdatafile.yaml
         pyats run job --trigger-datafile "http://path.to/triggerdatafile.yaml"
```

File: docs/userguide/harness/user/reference.rst [hunk 3]
```diff
@@ -325,6 +329,7 @@ Below you will find additional information on each of those arguments:
     .. code-block:: bash
 
          genie run /path/to/jobfile.py --subsection-datafile /path/subsection_datafile.yaml
+         genie run /path/to/jobfile.py --subsection-datafile "http://path.to/subsection_datafile.yaml"
```
Comment (tahigash) [inline docs/userguide/harness/user/reference.rst:332]: same
```suggestion
         pyats run job /path/to/jobfile.py --subsection-datafile /path/subsection_datafile.yaml
         pyats run jog /path/to/jobfile.py --subsection-datafile "http://path.to/subsection_datafile.yaml"
```

File: docs/userguide/harness/user/reference.rst [hunk 4]
```diff
@@ -348,6 +353,7 @@ Below you will find additional information on each of those arguments:
     .. code-block:: bash
 
          genie run /path/to/jobfile.py --pts-datafile /path/pts_datafile.yaml
+         genie run /path/to/jobfile.py --pts-datafile "hhtp://path.to/pts_datafile.yaml"
```
Comment (tahigash) [inline docs/userguide/harness/user/reference.rst:356]: same
```suggestion
         pyats run job /path/to/jobfile.py --pts-datafile /path/pts_datafile.yaml
         pyats run job /path/to/jobfile.py --pts-datafile "hhtp://path.to/pts_datafile.yaml"
```

**Review by tahigash — APPROVED** [review]
approved with comments

**Comment by tahigash** [comment]
@LukasMcClelland please address comments and get another approval.

File: docs/userguide/harness/user/reference.rst (hunk 1 above)
Comment (LukasMcClelland) [reply docs/userguide/harness/user/reference.rst:29]: yes, https://github.com/CiscoTestAutomation/pyats/pull/119/files

**Review by ThomasJRyan — APPROVED** [review]

**Pull request merged**